examples/branin/restore_state.py

```diff
@@ -0,0 +1,74 @@
+import os
+import logging
+import shutil
+
+from smac.facade.smac_facade import SMAC
+from smac.scenario.scenario import Scenario
+from smac.stats.stats import Stats
+from smac.runhistory.runhistory import RunHistory
+from smac.utils.io.traj_logging import TrajLogger
+
+"""
+This file runs SMAC and then restores the run with an extended computation
+budget. This will also work for runs that have crashed and are continued.
```

Comment:
runs is a bit ambiguous as it could mean SMAC optimization run or target algorithm run. How about "*SMAC* runs "?